_content/education.json

```diff
@@ -0,0 +1,201 @@
+[
```

Comment:
This file should generate from CMS, file to configure the CMS is on the `public/admin/config.yml`

Reply:
Ah, maybe I should put it in the `data` folder and create a function to fetch it from the sheet right?

Reply:
No, you should create a configuration for Netlify CMS.
Add the data from the dashboard.
Then this file will be created.